.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: ci
+
+on:
+  # Trigger the workflow on push to master or develop, except tag creation
+  push:
+    branches:
+      - 'master'
```

Comment:
There are still some "master" occurrences

Reply:
Thanks, that one sneakily got in.